Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -233,7 +245,7 @@ - (void)downloadObjectServer {
                        @"--no-save",
                        @"--no-package-lock",
                        @"install",
-                       [@"realm-object-server@" stringByAppendingString:desiredVersion]
+                       @"mongodb-stitch"
```

Comment:
The `desiredVersion` part here is still relevant; we don't want to just install whatever version happens to have the `latest` tag at the moment.